core/snippets/snippets/switchStatement.snippet

```diff
@@ -15,3 +15,12 @@ language: python
 match $1:
     $0
 ---
+
+language: r
+insertionScope: branch
+-
+case_when(
+    $0
+    .default = ${3:default_value}
```

Comment:
Why `default_value` as a placeholder?

Reply:
Just as a syntax indicator, to say that's where the default value goes

Reply:
Same here. Please remove.